### PR TITLE
Adds new Design System to Toggle component

### PR DESCRIPTION
**Pull request by martins87**
## Link to Issue
Closes: #3939 

##

### CODE REVIEW

**Review by CowMuon — APPROVED** [review]
Design to sign off on this.

**Comment by CowMuon** [comment]
Need design acceptance here @zakhap @meeshlin

**Comment by zakhap** [comment]
This looks good to me but needs 1 change:
- the animation speed for the toggle should be 100ms.

**Comment by martins87** [comment]
> This looks good to me but needs 1 change:
> 
> * [x]  the animation speed for the toggle should be 100ms.

Done.

File: packages/commonwealth/client/scripts/views/components/component_kit/cw_toggle.tsx [hunk 1]
```diff
@@ -40,7 +41,7 @@ export const CWToggle = (props: ToggleProps) => {
           className,
         },
         ComponentType.Toggle
-      )}
+      ).concat(` ${size}`)}
```
Comment (zakhap) [inline packages/commonwealth/client/scripts/views/components/component_kit/cw_toggle.tsx:44]: please add the `size` to the toggle style props above and just use the getClasses method.

**Review by zakhap — CHANGES_REQUESTED** [review]

File: packages/commonwealth/client/scripts/views/components/component_kit/cw_toggle.tsx (hunk 1 above)
Comment (martins87) [reply packages/commonwealth/client/scripts/views/components/component_kit/cw_toggle.tsx]: > please add the `size` to the toggle style props above and just use the getClasses method.

Done.

Also moved the new design component to new_designs folder:

image

**Pull request merged**